build.gradle

```diff
@@ -121,8 +121,10 @@ allprojects {
 		}
 
 		license {
-			header rootProject.file('HEADER')
+			header = rootProject.file('HEADER')
 			include '**/*.java'
+			exclude '**/lib/sponge/*.java'
+			exclude '**/lib/sponge/**/*.java'
```

Comment:
you know you can auto apply the sponge license headers in here to those files...

Reply:
already did

Reply:
actually sponge license headers were already applied so i didnt care